.gitignore

```
@@ -9,4 +9,5 @@ Package.resolved
 /*.sublime-workspace
 /.swiftpm
 .*.sw?
-__pycache__
\ No newline at end of file
+__pycache__
+.vscode/
```

```suggestion
/.vscode/
```

We can ignore the root one (auto generated by sourcekit-lsp), but the example one is currently built manually so I think we should keep that one for now

With this you should also be able to delete the root .vscode folder with this PR :)